packages/api/controllers/nomination.js

```diff
@@ -56,11 +57,19 @@ const updateNomination = async (req, res) => {
   try {
     const nomination = await db.Nomination.update(
       { status: req.body.status },
-      { where: { id } }
+      { where: { id }, returning: true }
     );
+    const updatedNom = nomination[1][0].dataValues
```

Comment:
I think we should add some protection surrounding the `nomination[1][0].dataValues` in the off chance that any of those are `undefined` or if `db.Nomination.update` returns a failure status resulting in a runtime error. 

Ping me if you want to work through this.